### ISSUE TITLE
Mutation to edit the Start time of a beacon

### ISSUE THREAD

**Issue by ItsAdityaKSingh** [CCExtractor/beacon-backend#98]
There is no mutation to change the start time of a beacon. A new mutation for editing start time could be introduced.

**Comment by ItsAdityaKSingh** [comment]
@aadibajpai For this, should I create a new mutation or edit the existing changeBeaconDuration for having the startsAt argument too?

**Comment by aadibajpai** [comment]
> @aadibajpai For this, should I create a new mutation or edit the existing `changeBeaconDuration` for having the `startsAt `argument too?

I think this is smart

**Comment by ItsAdityaKSingh** [comment]
I'm a little confused :) Should I edit the mutation itself is what you mean here?

**Comment by aadibajpai** [comment]
> I'm a little confused :) Should I edit the mutation itself is what you mean here?

yeah sorry should have been more specific, editing the existing mutation is better I think

**Comment by ItsAdityaKSingh** [comment]
Sure! Thanks for the advice :)
